web/index.html

```diff
@@ -495,7 +493,7 @@ <h2>Coding the MCTS algorithm</h2>
       We use a simple <code class="inline-code">Node</code> class to represent the nodes of the
       Monte-Carlo tree. Remember that a node is equivalent to a branch, and it represents a possible
       token (character) that could come next in the sequence of generated text. Each node carries
-      its own values for generation state, number of visits, node value, LLM token probability, and
```

Comment:
"own" here is important to keep, otherwise the meaning of the sentence subtly changes

Reply:
but its refers to its own